home-application/src/main/java/com/softserveinc/ita/homeproject/application/service/entity/EntitySpecificationService.java

```diff
@@ -1,4 +1,4 @@
-package com.softserveinc.ita.homeproject.application.service;
```

Comment:
I think that the package in which EntitySpecificationService and EntitySpecificationServiceImpl are located is better to name "Specification", since this name better reflects the essence of this service.